ecospheres_migrator/geonetwork.py

```diff
@@ -84,7 +88,11 @@ def info(self):
         return r.json()
 
     def authenticate(self):
-        r = self.session.post(f"{self.api}/info?_content_type=json&type=me")
+        r = self.session.post(f"{self.api}/info?_content_type=json&type=me", allow_redirects=False)
+        if r.is_redirect:
+            raise GeonetworkConnectionError(
+                f"Redirection détectée vers {r.headers['Location']}. Merci d'utiliser l'URL canonique du serveur."
+            )
         # don't abort on error here, it's expected
```

Comment:
I think I originally put that before we required auth. If you can get rid of it let's do it!

Reply:
Done. I'm keeping the redirect logic, since it will catch those errors more explicitly.